ui/admin/app/components/model-providers/ModelProviderBanner.tsx

```diff
@@ -0,0 +1,18 @@
+import { CircleAlert } from "lucide-react";
+
+import { TypographyP } from "~/components/Typography";
+
+export function ModelProviderBanner() {
+    return (
+        <div className="flex flex-row p-2 justify-start items-center w-[calc(100%-4rem)] rounded-sm bg-secondary relative overflow-hidden gap-2 max-w-screen-lg">
```

Comment:
```suggestion
        <div className="flex flex-row p-2 justify-start items-center pr-16 rounded-sm bg-secondary relative overflow-hidden gap-2 max-w-screen-lg">
```

Reply:
also is the goal to have the banner change **with** the `secondary` color? Or is the goal to make it a muted gray color? If the goal is to always be muted gray, then I'd recommend using `bg-muted` instead of `bg-secondary`

Reply:
My initial thought was for it to change with secondary but saw you mentioned the ShadCN alert and wondering if maybe that would look better! Will try it out! :+1: